src/NServiceBus.RabbitMQ.AcceptanceTests/NServiceBus.RabbitMQ.AcceptanceTests.csproj

```diff
@@ -1,286 +1,28 @@
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\TimeToBeReceived\When_TimeToBeReceived_has_expired_convention.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\TimeToBeReceived\When_TimeToBeReceived_has_not_expired.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\TimeToBeReceived\When_TimeToBeReceived_used_with_unobtrusive_mode.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\FakePromotableResourceManager.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\ImmediateDispatch\When_requesting_immediate_dispatch_using_scope_suppress.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\ImmediateDispatch\When_requesting_immediate_dispatch_with_at_least_once.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\ImmediateDispatch\When_requesting_immediate_dispatch_with_at_most_once.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\ImmediateDispatch\When_requesting_immediate_dispatch_with_exactly_once.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\ImmediateDispatch\When_sending_inside_ambient_tx.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\Issue_2481.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\When_receiving_with_dtc_enabled.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\When_receiving_with_native_multi_queue_transaction.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\When_receiving_with_the_default_settings.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Tx\When_sending_within_an_ambient_transaction.cs" />
-    <Compile Include="App_Packages\NSB.AcceptanceTests.6.2.0\Versioning\When_multiple_versions_of_a_message_is_published.cs" />
-    <Compile Include="DelayedDelivery\When_deferring_a_message_longer_than_allowed_maximum.cs" />
-    <Compile Include="TestSuiteConstraints.cs" />
-    <Compile Include="When_using_a_custom_message_id_strategy_that_returns_an_invalid_message_id.cs" />
-    <Compile Include="ConfigureEndpointRabbitMQTransport.cs" />
-    <Compile Include="TaskEx.cs" />
-    <Compile Include="When_receiving_a_reply_that_contains_a_legacy_callback_header.cs" />
-    <Compile Include="When_the_message_contains_a_legacy_callback_header.cs" />
-    <Compile Include="When_publishing_message_implementing_interface_in_direct_topology.cs" />
-    <Compile Include="When_using_direct_routing.cs" />
-    <Compile Include="When_the_broker_connection_is_lost.cs" />
-    <Compile Include="When_using_a_custom_message_id_strategy.cs" />
-    <Compile Include="When_scaling_out_senders_that_uses_callbacks.cs" />
-    <Compile Include="When_scaling_out_subscribers.cs" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
```

Comment:
For my own education, what is this for? (I think you mentioned it before but I've forgotten.)